src/DotNetWorker/Context/Features/IBindingFeature.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Functions.Worker.Context.Features
+{
+    internal interface IBindingFeature
```

Comment:
We just talked about an `IFunctionBindingsFeature` yesterday. Should we call this `IModelBindingFeature` or something else to disambiguate? I know you don't like "Converters" :-)

Reply:
I'll make the change now to avoid confusion. Still keeping those types internal, so we can go through another API and naming review at a later time.